association/privacy.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: "Membership"
+---
+
+# Privacy Policy
+
+
+**Website**
```

Comment:
This was confusing to me, since I was reading it about the website but it seems to be more about the conference.

I'd recommend making a dedicated section for the conference (separate from NRSE membership), and subsections for a) attendees b) presenters.  Then, say what happens for each of these people.

Reply:
There were several sentences left in the wrong place, left over from an edit